src/test/java/com/google/roomies/database/FirebaseDatabaseTest.java

```diff
@@ -99,6 +111,38 @@ public void testAddListingAsMap_addsSingleListingToFirestore() throws Exception
     verify(collectionMock, Mockito.times(1)).add(listingData);
   }
 
+  @Test
+  public void testaddCommentAsMapToListing_addsSingleCommentToFirestore() throws Exception {
+    when(docSnapshotMock.exists()).thenReturn(true);
+    String listingId = "7YDcsjQOTzVoUxeXiysT";
```

Comment:
use testListingId rather than 7YDcsjQOTzVoUxeXiysT. It avoid confusion of thinking may be 7YDcsjQOTzVoUxeXiysT is real id.